cms/djangoapps/contentstore/views/item.py

```diff
@@ -846,7 +849,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
         "actions": xblock_actions,
-        "explanatory_message": explanatory_message
+        "explanatory_message": explanatory_message,
+        "enable_proctored_exams": enable_proctored_exams,
+        "is_proctored_enabled": xblock.is_proctored_enabled if xblock.category == 'sequential' else False,
+        "is_time_limited": xblock.is_time_limited if xblock.category == 'sequential' else False,
+        "default_time_limit_mins": getattr(
+            xblock,
+            'default_time_limit_mins', '00:00'
```

Comment:
I'll look into this, but the client JS uses a HH:MM notation in the presentation control, as per the mockup from UX. We could do the HH:MM <--> integer minutes in JavaScript, tho.